src/bulkDataAccess.ts

```diff
@@ -47,7 +48,7 @@ export interface BulkDataAccess {
      * https://hl7.org/Fhir/uv/bulkdata/export/index.html#bulk-data-delete-request
      * @param jobId - Id of the job you would like to cancel the export request for
```

Comment:
Need to add comments throughout if there are @param fields

Reply:
updated